sdk/core/core-tracing/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Provides low-level interfaces and helper methods for tracing in Azure SDK",
   "sdk-type": "client",
   "main": "dist/index.js",
-  "module": "dist-esm/index.js",
+  "module": "dist-esm/lib/index.js",
```

Comment:
good catch! 👍

Reply:
This change was only necessary since I added tests. Since they weren't there before TypeScript didn't just flattened the files into `dist-esm` :)